packages/datadog-plugin-jest/src/index.js

```diff
@@ -156,34 +156,31 @@ class JestPlugin extends Plugin {
       if (!this.config.isAgentlessEnabled) {
         return
       }
-      const store = storage.getStore()
       const childOf = getTestParentSpan(this.tracer)
       const testSessionSpanMetadata = getTestSessionCommonTags(command, this.tracer._version)
 
-      const testSessionSpan = this.tracer.startSpan('jest.test_session', {
+      this.testSessionSpan = this.tracer.startSpan('jest.test_session', {
```

Comment:
The PR description explains what this does, but not really why. We should never attach a span to an object that might end up in multiple asynchronous contexts.